### PR TITLE
Add libzip 1.9.2

### PR DESCRIPTION
**Pull request by bucanero**
the latest libzip from https://libzip.org/

### CODE REVIEW

**Comment by sharkwouter** [comment]
Thanks for the PR @bucanero! It seems the automated build fails because xz is not available. Could you add it to the apk add steps found in this file?: https://github.com/pspdev/psp-packages/blob/master/.github/workflows/build.yml

**Comment by bucanero** [comment]
hi @sharkwouter , I have added `xz` based on your feedback

Btw, libzip also provides a `.tar.gz` release, so if needed/preferred I can switch to https://libzip.org/download/libzip-1.9.2.tar.gz and update the sha256 hash.

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -104,7 +104,7 @@ jobs:
     - uses: actions/download-artifact@v3
     - name: Install dependencies
       run: |
-        apk --update add build-base bash gpgme-dev libarchive-tools libarchive-dev libtool doxygen libcrypto1.1
+        apk --update add build-base bash gpgme-dev libarchive-tools libarchive-dev libtool doxygen libcrypto1.1 xz
```
Comment (sharkwouter) [inline .github/workflows/build.yml:107]:
```suggestion
        apk --update add build-base bash gpgme-dev libarchive-tools libarchive-dev libtool doxygen libcrypto1.1
```

This one is not needed.

**Review by sharkwouter — CHANGES_REQUESTED** [review]
It seems the automated build is still failing.

**Comment by bucanero** [comment]
> It seems the automated build is still failing.

my bad, I committed a fix so it can change directory as expected `cd "$pkgname-$pkgver"` (`cd libzip-1.9.2`)

**Comment by bucanero** [comment]
another fix, seems like `prepare()` can't be empty

**Comment by sharkwouter** [comment]
You can probably just remove prepare

**Comment by bucanero** [comment]
> You can probably just remove prepare

`prepare() { }` removed. Let's see if it works. 😄

**Review by sharkwouter — CHANGES_REQUESTED** [review]
Nice, it builds now. Looks good too, thanks!

One nitpick, coud you make sure the .pc file does not contain the full path of where it was built? Here is an example: https://github.com/pspdev/psp-packages/blob/4135278ee8610747a8bf182ba9d5940ea1d2f515/eigen3/PSPBUILD#L15

You'll have to patch libzip.pc.in like that. Currently it looks like this:
```
prefix=/__w/psp-packages/psp-packages/libzip/pkg/libzip/psp
exec_prefix=${prefix}
bindir=${prefix}/bin
libdir=${prefix}/lib
includedir=${prefix}/include
``
Should be:
```
prefix=${PSPDEV}/psp
exec_prefix=${prefix}
bindir=${prefix}/bin
libdir=${prefix}/lib
includedir=${prefix}/include
```

**Comment by bucanero** [comment]
thanks for the feedback and tips on the `.pc` file. 👍 

I've re-added `prepare()` and added a patch to update `libzip.pc.in` as needed. Everything should work as expected now. 😄

**Comment by sharkwouter** [comment]
Thanks! Looks good to me

**Pull request merged**